flexget/utils/tools.py

```diff
@@ -160,12 +276,12 @@ def merge_dict_from_to(d1, d2):
                     merge_dict_from_to(d1[k], d2[k])
                 elif isinstance(v, list):
                     d2[k].extend(copy.deepcopy(v))
-                elif isinstance(v, (basestring, bool, int, float, type(None))):
```

Comment:
Because i was told that basestring should not be used anymore and i spotted this here.